builds/linux-storage.pkrvars.hcl.example

```diff
@@ -141,7 +141,7 @@ vm_disk_lvm = [
       },
       {
         name = "lv_audit",
-        size = 4096,
+        size = -1,
```

Comment:
I can add 
`"size = -1" can also be used for the last partition (Logical Volume) of "vm_disk_lvm" so that it fills the remaining space of the Volume Group.`  
at the end of :  
https://vmware-samples.github.io/packer-examples-for-vsphere/getting-started/configuration/#storage-customization

Reply:
commit added to the PR